### PR TITLE
Add jvm testNG sample

### PR DESCRIPTION
**Pull request by Zuretto**
### Outline
Added TestNG sample to the project.
Included removing TestNG runner Stack frames in the source code.

### Issue
#1562
______________________________________
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/main/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

**Comment by Zuretto** [comment]
Hi @robstoll, I was not sure if removing TestNG runner stack frames should be added to the project. I thought it would be good to add it in this PR when I was searching the repository for kotest usages.

Please let me know if it should be part of the PR or if I should exlude it completely or create new PR for that one change.

File: atrium-core/src/jvmMain/kotlin/ch/tutteli/atrium/reporting/erroradjusters/impl/RemoveRunnerFromAtriumErrorImpl.kt [hunk 1]
```diff
@@ -11,6 +11,7 @@ actual class RemoveRunnerFromAtriumErrorImpl : FilterAtriumErrorAdjuster(), Remo
                 !it.className.startsWith("org.jetbrains.spek") &&
                 !it.className.startsWith("org.spekframework.spek2") &&
                 !it.className.startsWith("io.kotest") &&
+                !it.className.startsWith("org.testng") &&
```
Comment (robstoll) [inline atrium-core/src/jvmMain/kotlin/ch/tutteli/atrium/reporting/erroradjusters/impl/RemoveRunnerFromAtriumErrorImpl.kt:14]: Thanks, that's perfect :slightly_smiling_face: :+1:

**Review by robstoll — APPROVED** [review]

File: misc/tools/atrium-samples-test/build.gradle.kts [hunk 2]
```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 //TODO 1.2.0 include ...-kotlin-test again once we use IR for JS
-val samples = listOf("junit5", "spek", "kotest")//, "js-kotlin-test", "multiplatform-kotlin-test")
+val samples = listOf("junit5", "spek", "kotest", "testng")//, "js-kotlin-test", "multiplatform-kotlin-test")
```
Comment (robstoll) [inline misc/tools/atrium-samples-test/build.gradle.kts:6]: you also need to add it in settings.gradle.kts in this directory

**Review by robstoll — CHANGES_REQUESTED** [review]

File: misc/tools/atrium-samples-test/build.gradle.kts (hunk 2 above)
Comment (Zuretto) [reply misc/tools/atrium-samples-test/build.gradle.kts:6]: True, added. Thanks for the hint!

**Comment by robstoll** [comment]
@Zuretto thanks for your first contribution to Atrium :+1: 

How about another one?

**Pull request merged**